src/releaseNotesProvider.ts

```diff
@@ -128,7 +132,9 @@ async function showReleaseNotesImpl(
 
     return isNewerVersion
       ? makeRight(cleanVersion)
-      : makeLeft("do not show release notes for an older version");
+      : makeLeft(
+          "do not show release notes for an older version/already seen version"
```

Comment:
```suggestion
          `do not show release notes for switching to the older or same version, previous version: ${previousVersion}, current version: ${currentVersion}`
```

What do you think about this? I personally think "same version" makes more sense than "already seen version" (if I understand correctly). Also, it would be helpful if there's version information in the message.

Reply:
Release notes should be shown only once for particular version (vs code keeps track of the newest one) and not shown for an older version than was already seen.
This message (it's more a debug one) will be logged to the output tab when any of aforementioned conditions are false:
- release notes were showed for current version
- user is using version older than the last one for which notes were showed

Reply:
Ah ok, so something like

```
not showing release notes since they've already been seen for your current version
```

???

Reply:
👍 to @ckipp01 's comment, my suggestion was too much related to the internal behavior that shouldn't be exposed to  end users.